packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -89,6 +90,7 @@ export class FacetSearch {
     this.props.controller.facetSearch.select(
       value as CategoryFacetSearchResult
     );
+    this.props.controller.text = '';
```

Comment:
Should we include cypress tests in this PR? Sometimes I see we are writing them, sometimes it is QA, so not sure?

Reply:
Might have slipped by me before, but why is the controller's state modified directly here, in the FacetSearchController:

```
public set text(text: string) {
  -> this.state.facetSearchQuery = text; 
  this.facetSearch.updateText(text);
  this.facetSearch.search();
}
```
Is there an action missing on the headless controller?

Reply:
Ohh I was confused, it's actually a state in Stencil, makes sense now 😄